pootle/apps/pootle_app/forms.py

```diff
@@ -93,8 +95,13 @@ def clean_code(self):
         return self.cleaned_data['code'].strip()
 
     def save(self, commit=True):
+        created = bool(not self.instance.pk)
         project = super(ProjectForm, self).save(commit=commit)
         project.config["pootle_fs.fs_type"] = self.cleaned_data["fs_plugin"]
+        if created and self.cleaned_data["fs_plugin"] == "localfs":
```

Comment:
What if an already existing project changed its plugin to `localfs`. I would just drop the `created` bit.

Reply:
if we do that it will overwrite any custom path

Reply:
Sure, but that is what we want, isn't it?

Reply:
i mean it would overwrite an already configured project

Reply:
mmm, you are right. But the first part of my initial comment in this thread still stands.

Reply:
if you change the fs_type then it would be kinda up to you to update the url too. This is just setting a default